darwin/dataset/remote_dataset_v2.py

```diff
@@ -842,3 +862,136 @@ def register_multi_slotted(
                 print(f"  - {item}")
         print(f"Reistration complete. Check your items in the dataset: {self.slug}")
         return results
+
+
+def _find_files_to_upload_merging(
```

Comment:
I'll rename to `_find_files_to_upload_with_merging` - Since the function is used for DICOM series & multi-channel items as well, I'd avoid using `multislot` in the name

Reply:
Addressed in [comment below](https://github.com/v7labs/darwin-py/pull/923/files#r1751825121), there are better, more generic names I'll use